app.js

```diff
@@ -0,0 +1,50 @@
+$(document).ready((e) => {
+
+    const key = myL3.apiKeys.openweathermapKey;
+
+    getData = () => {
+        $('#import').hide();
+        if (navigator.geolocation) {
+
+            options = {
+                enableHighAccuracy: true,
+                timeout: 30000,
+                maximumAge: 0
+            };
+
+            error = (err) => {
+                console.log(err);
+                $('#err').html(`Cannot retrieve location info, \
+                <br>Reason: <strong>${err.message}</strong>`);
+                if(!navigator.onLine)
+                    alert('Internet connection not avaiable');
+            }
+
+            getWeatherData = (position) => {
+                console.log('Coordinates: ', position);
+                const lat = position.coords.latitude;
+                const lon = position.coords.longitude;
+                const location = JSON.stringify(position.coords);
+
+                $('#coords').html(`<h3>Location data</h3>Latitude: ${lat} <br> Longitude: ${lon}`);
+
+                $.getJSON(`https://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${lon}&APPID=${key}`, (weatherData) => {
+                    console.log('Weather JSON', weatherData);
+                    const wJSON = JSON.stringify(weatherData);
+                    $('#city').html(`Country: ${myL3.countryNames[weatherData.sys.country]} <br> City: ${weatherData.name} <br><br>`)
+                    $('#wData').html(`<h3>Weather Data:</h3><p> ${wJSON}</p>`);
+                })
+                .fail(() => {
+                    console.log('Weather data not available for this region.');
+                });
+
+                // we can also get weather forecast for next 5 days using following link:
+                // http://api.openweathermap.org/data/2.5/forecast?lat=${lat}&lon=${lon}&APPID=${key}
+            }
+
+            navigator.geolocation.getCurrentPosition(getWeatherData, error, options);
+        }
+        else
+            console.log('User\'s browser doesn\'t support geolocation');
```

Comment:
Please put back the enclosing curly braces on this `else` clause. And please also add braces around the if test on line 19.

Reasoning: it's easy to introduce bugs and misread code written with C-style if/else blocks that omit the curly braces. For example see https://www.dwheeler.com/essays/apple-goto-fail.html for an analysis of a bug caused in part by an `else` clause without braces. Section 3.3.

For our projects, we use ESLint to enforce the use of curly braces. We use the `curly: all` rule:

- https://github.com/fluid-project/eslint-config-fluid/blob/master/.eslintrc.json#L21
- https://eslint.org/docs/rules/curly

We've got a package `eslint-config-fluid` (linked above) that we use in our projects. Once we've merged in this pull request, I will make a pull request to add use of this package. So that we can use it to check against our standard JavaScript rules.

Reply:
I will take care of this thing in future. Thanks a lot.